### PR TITLE
Added Deleted monitors section to monitor settings page

### PR DESCRIPTION
**Pull request by SimoneT-DD**
### What does this PR do?
The Monitors Settings page is now not only about tag policies. A new tab has been added to restore recently deleted monitors. The doc has been updated describing the topic that can be accessed in the monitor settings page.

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/monitors/settings/_index.md [hunk 1]
```diff
@@ -48,15 +56,27 @@ To make a tag optional but require that monitors with the tag use specific value
...
+### Permissions
 
 To configure monitor tag policies, you must be assigned a role with the `MONITOR_CONFIG_POLICY_WRITE_PERMISSION` permission.
 
-For more information, see [Role Based Access Control][1] and [Role Permissions][2].
+For more information, see [Role Based Access Control][2] and [Role Permissions][3].
+
+
+## Deleted monitors
+Deleted Monitors allow you to restore recently deleted Datadog monitors. Monitors are retained for 7 days before being permanently deleted.
```
Comment (estherk15) [inline content/en/monitors/settings/_index.md:67]:
```suggestion
Monitors are retained for 7 days before being permanently deleted. Restore recently deleted Datadog monitors:
```

File: content/en/monitors/settings/_index.md [hunk 2]
```diff
@@ -48,15 +56,27 @@ To make a tag optional but require that monitors with the tag use specific value
...
+
+### Restore deleted monitor
```
Comment (estherk15) [inline content/en/monitors/settings/_index.md:69]:
```suggestion
```

File: content/en/monitors/settings/_index.md [hunk 3]
```diff
@@ -48,15 +56,27 @@ To make a tag optional but require that monitors with the tag use specific value
...
+1. Navigate to the **Monitors** > **Settings** page.
```
Comment (estherk15) [inline content/en/monitors/settings/_index.md:70]:
```suggestion
1. Navigate to the [**Monitors** > **Settings**][1] page.
```

**Review by estherk15 — APPROVED** [review]
Thanks for fixing this @SimoneT-DD , left a few suggestions. When it's ready, I can publish!

File: content/en/monitors/settings/_index.md [hunk 4]
```diff
@@ -12,17 +12,25 @@ further_reading:
...
 
 <div class="alert alert-warning">After set up, tag policies apply to <strong>all</strong> Datadog monitors</div>
 
 - To create a new monitor, it must adhere to your organization's tag policies.
 - Existing monitors that violate your organization's tag policies continue to provide alerts and notifications, but must be updated to match the tag policies before you can modify other settings.
 
-## Configure monitor tag policies
+### Configure monitor tag policies
 
 1. Navigate to the **Monitors** > **Settings** page.
```
Comment (SimoneT-DD) [inline content/en/monitors/settings/_index.md:32]:
```suggestion
1. Navigate to the [**Monitors** > **Settings**][1] page.
```

File: content/en/monitors/settings/_index.md [hunk 5]
```diff
@@ -48,15 +56,26 @@ To make a tag optional but require that monitors with the tag use specific value
...
+### Permissions
 
 To configure monitor tag policies, you must be assigned a role with the `MONITOR_CONFIG_POLICY_WRITE_PERMISSION` permission.
 
-For more information, see [Role Based Access Control][1] and [Role Permissions][2].
+For more information, see [Role Based Access Control][2] and [Role Permissions][3].
+
+
+## Deleted monitors
+Monitors are retained for 7 days before being permanently deleted. Restore recently deleted Datadog monitors:
```
Comment (estherk15) [inline content/en/monitors/settings/_index.md:67]:
```suggestion
Monitors are retained for 7 days before being permanently deleted. To restore recently deleted Datadog monitors:
```

**Review by estherk15 — APPROVED** [review]

**Pull request merged**